library.js

```diff
@@ -0,0 +1,90 @@
+// declare empty array for library
+let myLibrary = [];
+let i = 0;
+//object constructor
+function Book(Title, Author, Pages, Read) {
+  this.Title = Title;
+  this.Author = Author;
+  this.Pages = Pages;
+  this.Read = Read;
+}
+// function to add a new book to library
+function addBookToLibrary(Title, Author, Pages, Read) {
+  let book = new Book(Title, Author, Pages, Read);
+  myLibrary.push(book);
+  displayBooksOnPage();
+}
+//for displaying the books in library
+function displayBooksOnPage() {
```

Comment:
Break this function into small helper functions such as `createCard, createRemoveButton, createReadButton`